pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -820,6 +843,39 @@ public String generateUniqueProducerName() {
     public AuthenticationService getAuthenticationService() {
         return authenticationService;
     }
+    
+    private int getLookupRequestPermits() {
+        int pendingLookupRequest = pulsar.getConfiguration().getMaxConcurrentLookupRequest();
+        try {
+            Optional<Map<String, String>> data = dynamicConfigurationCache.get(LOOKUP_THROTTLING_PATH);
```

Comment:
Either that, or just use the value from config file as it is (like any other config parameter) and then we can open a separate issue to discuss the broader dynamic config approach.